README.md

```diff
@@ -41,3 +41,31 @@ be found.
 
 You will need `mdbook` version `>= 0.2`. `linkcheck` will be run automatically
 when you run `mdbook build`.
+
+## How to fix toolstate failures
+
+1. You will get a ping from the toolstate commit. e.g. https://github.com/rust-lang-nursery/rust-toolstate/commit/8ffa0e4c30ac9ba8546b7046e5c4ccc2b96ebdd4
+
+2. The commit contains a link to the PR that caused the breakage. e.g. https://github.com/rust-lang/rust/pull/64321
+
+3. If you go to that PR's thread, there is a post from bors with a link to the CI status: https://github.com/rust-lang/rust/pull/64321#issuecomment-529763807
+
+4. Follow the check-azure link to get to the Azure website for that build: https://dev.azure.com/rust-lang/e71b0ddf-dd27-435a-873c-e30f86eea377/_build/results?buildId=7780
+
+5. There will be approximately 1 billion different jobs for the build. They are for different configurations and platforms. The rustc-guide build only runs on the Linux x86_64-gnu-tools job. So click on that job in the list, which is about 60% down in the list.
+
+6. Click the Run build step in the job to get the console log for the step.
+
+7. Click on the log and Ctrl-f to get a search box in the log
+
+8. Search for rustc-guide. This gets you to the place where the links are checked. It is usually ~11K lines into the log
+
+9. Look at the links in the log near that point in the log
+
+10. Fix those links in the rustc-guide (by making a PR in the rustc-guide repo)
+
+11. Make a PR on the rust-lang/rust repo to update the rustc-guide git submodule in src/docs/rustc-guide
```

Comment:
IMO: It'd be nice that we have the guidance to make a PR something like this here: https://github.com/rust-lang/rust-clippy/blob/master/CONTRIBUTING.md#fixing-build-failures-caused-by-rust

Reply:
@JohnTitor :+1:, I've just merged this but we could work on something like what you're suggesting on a different PR. Feel free to provide a PR for that or open an issue.